perfkitbenchmarker/cloudstack/__init__.py

```diff
@@ -1,13 +0,0 @@
-# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
Looks like PyCharm do it automaticaly in refactoring option - but this file is available in pkb/providers/cloudstack/__init__.py